packages/thirdweb/benchmarks/encode-tx.ts

```diff
@@ -126,8 +130,7 @@ group("encode transfer (cold cache)", () => {
 
   bench("viem", async () => {
     viem.encodeFunctionData({
-      abi: ABI,
-      functionName: "transfer",
+      ...transfer_viem,
```

Comment:
hmmm that's not really cold cache is it?

Reply:
this is equivalent to [`transfer` on the thirdweb bench](https://github.com/thirdweb-dev/js/pull/2516/files/f90ffdb4ff9a71be8ca031d6d0c7514de865f14b#diff-3ba235fd736bcbaf2740c128e25e522f307f933328aba1fe3e8126b8a4b3eacbR112) in this suite.

Reply:
right but one is compiled ahead of time, the other is computed at runtime. When we say cold cache, generally that means you start from nothing. In your case the user would have to already have done some work right?

Reply:
From the perspective of benchmarking, I'd expect "cold cache" to refer to the invalidation of internal caches where the computation (between I/O) is happening. Doesn't matter how you generate your input vectors. If we are "starting from nothing", we shouldn't even load in the compiled vectors. ;P

Reply:
maybe a better word is 'cold start'? Don't think its fair to do the runtime prepare work in both benchmarks here.